items_orders.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE items (
+  id SERIAL PRIMARY KEY,
+  name text,
+  price int,
```

Comment:
Could use the property "float8" for the data type of price to show the price to two decimal places.